scene/choose-camera-controller/README.md

```diff
@@ -14,18 +14,18 @@ The application loads with the default globe camera controller. To rotate and fi
 
 ## How it works
 
-1. Create an instance of a class extending `CameraController`: `GlobeCameraController`, `OrbitLocationCameraController`, `OrbitGeoElementCameraController`
+1. Create an instance of a class extending `CameraController`: `GlobeCameraController`, `OrbitLocationCameraController`, `OrbitGeoElementCameraController`.
```

Comment:
I'm not too sure about this description, since our sample class doesn't extend Camera controller: we instantiate a new instance of each within the class.

Reply:
This just needs rephrasing. I think what it's trying to say is that either `GlobeCameraController`, `OrbitLocationCameraController`, or `OrbitGeoElementCameraController` are classes that extend `CameraController`, and thus we need to create an instance of one of these three. Any suggestions?

Reply:
how about:

> Create new instances of the following classes which extend from `CameraController`:

Reply:
Sounds good to me!